src/cryptography/x509/base.py

```diff
@@ -21,9 +21,9 @@
     x448,
 )
 from cryptography.hazmat.primitives.asymmetric.types import (
+    CERTIFICATE_ISSUER_PUBLIC_KEY_TYPES,
+    CERTIFICATE_PRIVATE_KEY_TYPES,
```

Comment:
This doesn't re-export so may break people. Do we want to re-export these symbols with the old names?